components/FlexibleHeader/src/MDCFlexibleHeaderView.m

```diff
@@ -1724,75 +1723,28 @@ - (void)stopForwardingTouchEventsForView:(UIView *)view {
   [_forwardingViews removeObject:view];
 }
 
-- (void)setMinimumHeight:(CGFloat)minimumHeight {
-  _hasExplicitlySetMinHeight = YES;
-  if (_minimumHeight == minimumHeight) {
-    return;
-  }
-
-  _minimumHeight = minimumHeight;
-
-  if (_minimumHeight > _maximumHeight) {
-    [self setMaximumHeight:_minimumHeight];
-  } else {
-    [self fhv_updateLayout];
-  }
+- (CGFloat)minimumHeight {
+  return self.minMaxHeight.minimumHeight;
 }
 
-- (void)setMaximumHeight:(CGFloat)maximumHeight {
-  _hasExplicitlySetMaxHeight = YES;
-  if (_maximumHeight == maximumHeight) {
-    return;
-  }
-
-  _maximumHeight = maximumHeight;
-
-  [self fhv_adjustTrackingScrollViewInsets];
+- (void)setMinimumHeight:(CGFloat)minimumHeight {
+  self.minMaxHeight.minimumHeight = minimumHeight;
+}
 
-  if (_maximumHeight < _minimumHeight) {
-    [self setMinimumHeight:_maximumHeight];
-  } else {
-    [self fhv_updateLayout];
```

Comment:
why are these calls not needed anymore? There seemed to be more than min/max calculations in this method. For example what happens if the min is set to larger than the current value.

Reply:
This, and all other logic in this file, has been moved to the minmax height object as a refactoring. Logic-speaking this change is expected to be a no-op.